File: C/Day_5.c
//Relational Operators/ Comparison Operators

#include <stdio.h>
int main(){
    int num1, num2;
    printf("Enter num1: ");
    scanf("%d", &num1);
    printf("Enter num2: ");
    scanf("%d", &num2);
    printf("Equal? %d\n", num1==num2);
    printf("Not Equal? %d\n", num1!=num2);
    printf("Less than? %d\n", num1<num2);
    printf("Greater than? %d\n", num1>num2);
    printf("Less than or Equal? %d\n", num1<=num2);
    printf("Greater than or Equal? %d\n", num1>=num2);
    return 0;
}

File: Java/Day_5.java
// Relational Operators/ Comparison Operators

import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int num1 = scan.nextInt();
        int num2 = scan.nextInt();
        System.out.println("Equal? " + (num1==num2));
        System.out.println("Not Equal? " + (num1!=num2));
        System.out.println("Less than? " + (num1<num2));
        System.out.println("Greater than? " + (num1>num2));
        System.out.println("Less than or equal? " + (num1<=num2));
        System.out.println("Greater than or equal? " + (num1>=num2));
    }
}
